src/modules/network/utils/api/index.js

```diff
@@ -22,7 +22,7 @@ export const getNetworkStatus = ({ baseUrl }) =>
     path: httpPaths.networkStatus,
   });
 
-const getServiceUrl = ({ name, address = 'http://165.227.246.146:9901' }) => {
+const getServiceUrl = ({ name, address = 'http://mainnet-service.lisk.com' }) => {
```

Comment:
this can come from the configuration object 

```suggestion
const getServiceUrl = ({ name, address = networks[networkKeys.mainnet].serviceUrl }) => {
```